cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -4,9 +4,20 @@
 
 {% set form = settings.form %}
 
+{% set messages = get_messages(request) %}
+
 {# JS-disabled result handling #}
 
 <div class="o-feedback block">
+    {% if messages.__len__() %}
+    <!-- JS-disabled Form POST return -->
+    <div class="u-mb15">
```

Comment:
What's the `u-mb15` necessary for? I don't see a change when I remove it. Am I missing something?

Reply:
In fact, this whole div can go and it can be just:

```
{% if messages.__len__() %}
    <!-- JS-disabled Form POST return -->
    {% for message in messages %}
        {{ notification.render( message.tags.split(' ')[0], true, message.message ) }}
    {% endfor %}
    {% else %}
    
    …
```

Reply:
Looks like the `u-mb15` used in the JS-version is so that the form has padding at the top in the space where the notification occupies before it appears, which is a different structure to the non-js version.